### PR TITLE
Add Explicit Names For CI

### PR DESCRIPTION
**Pull request by tateexon**

### CODE REVIEW

File: .github/workflows/chaos.yml [hunk 1]
```diff
@@ -11,8 +11,8 @@ on:
         type: string
 
 jobs:
-  build_custom_cl_image:
-    name: Build custom CL image
+  E2E_Chaos_Build_Custom_Chainlink_Image:
```
Comment (krebernisak) [inline .github/workflows/chaos.yml:14]: Do these actually show anywhere now that we have explicit names for all?

I'd prefer we leave them lowercase, to follow the rest of `yml` workflow style, but we should make sure to add an explicit name for each.

File: .github/workflows/gauntlet.yml [hunk 2]
```diff
@@ -26,9 +24,10 @@ jobs:
           nix develop -c yarn install --frozen-lockfile
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (krebernisak) [inline .github/workflows/gauntlet.yml:26]: Please also remove the `NPM_TOKEN` setup from workflows, as `gautlet-core` should now be public so this shouldn't be necessary anymore.

File: .github/workflows/golangci-lint.yml [hunk 3]
```diff
@@ -3,14 +3,16 @@ name: golangci-lint
 on:
   push:
     branches:
-      - auto
-      - try
-      - rollup
+      - develop
+      - main
+    paths:
+      - '**.go'
```
Comment (krebernisak) [inline .github/workflows/golangci-lint.yml:9]: Will this only run if `'**.go'` files are touched?

If that's the case, then we can't enable this as a global check for develop branch. Consider removing this filter, for now, so we enable this check globally.

**Review by krebernisak — APPROVED** [review]

**Review by krebernisak — CHANGES_REQUESTED** [review]

File: .github/workflows/gauntlet.yml [hunk 4]
```diff
@@ -1,13 +1,11 @@
...
 on:
   push:
-    branches:
-      - develop
-  pull_request:
 
 jobs:
-  build:
+  gauntlet_build:
+    name: Guantlet Build
```
Comment (aalu1418) [inline .github/workflows/gauntlet.yml:8]:
```suggestion
    name: Gauntlet Build
```

**Review by aalu1418 — CHANGES_REQUESTED** [review]

File: .github/workflows/gauntlet.yml (hunk 4 above)
Comment (tateexon) [reply .github/workflows/gauntlet.yml]: 🤦

**Review by aalu1418 — APPROVED** [review]

File: .github/workflows/chaos.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-name: E2E Chaos Tests
+name: e2e_chaos_tests
```
[the remark this answers is not in the document]
Comment (krebernisak) [reply .github/workflows/chaos.yml:1]: Why the discrepancy from Solana? What was the problem when using nix there?

[the remark this answers is not in the document]
Comment (tateexon) [reply .github/workflows/chaos.yml:1]: Nix on Solana had a 1 hour cache issue for the first call using it for every run. We don't have that for Terra. When we solve that cache issue for nix on Solana have it use nix as well.

**Review by aalu1418 — APPROVED** [review]

File: .golangci.yml [hunk 6]
```diff
@@ -37,7 +37,7 @@ linters-settings:
       - name: error-return
       - name: error-strings
       - name: error-naming
-      - name: exported
+      # - name: exported
```
Comment (krebernisak) [inline .golangci.yml:40]: We should document all Go exported (public) types

**Review by krebernisak — APPROVED** [review]

**Pull request merged**